scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/TreeExtensions.scala

```diff
@@ -36,8 +38,8 @@ trait TreeExtensions {
       TreeOps.exists(a)(f)
 
     @inline
-    def contains[F[_ <: Tree]](toFind: Tree)(implicit conv: Tree => F[Tree],
-                                             eqEv: Equal[F[Tree]]): Boolean =
+    def contains[F[x <: Tree] <: TreeEquality[x]](toFind: Tree)(implicit conv: Tree => F[Tree],
```

Comment:
The reason I make it lowercase is because `x` is not a regular type parameter, it's only scoped inside the type param list. I saw `x` used in the same situation here: https://github.com/OlivierBlanvillain/monadic-html/blob/de78d4b147e39f5fdec300cce4c2b68e3596bf3a/monadic-html/src/main/scala/scala/xml/xml.scala#L180-L182